yaml/pois.yaml

```diff
@@ -2100,6 +2138,18 @@ filters:
       <<: *output_properties
       kind: sanitary_dump_station
 
+  - filter: {highway: elevator}
+    min_zoom: 16
```

Comment:
To `min_zoom` 17, please. This is an "indoor" feature.

Reply:
OK. Updated in 1e6b874.